backend/lcfs/web/api/other_uses/repo.py

```diff
@@ -76,6 +77,7 @@ async def get_table_options(self, compliance_period: str) -> dict:
                     )
                 )
             )
+            .unique()
```

Comment:
More unique

Reply:
The same here, this query is complex and you are required to use unique()